packages/web-app-files/src/views/Personal.vue

```diff
@@ -198,16 +204,73 @@ export default {
   },
 
   methods: {
-    ...mapActions('Files', ['loadIndicators', 'loadPreview']),
+    ...mapActions('Files', ['loadIndicators', 'loadPreview'], 'showMessage'),
     ...mapMutations('Files', [
       'SELECT_RESOURCES',
       'SET_CURRENT_FOLDER',
       'LOAD_FILES',
       'CLEAR_CURRENT_FILES_LIST',
-      'UPDATE_CURRENT_PAGE'
+      'REMOVE_FILE_FROM_SEARCHED'
     ]),
     ...mapMutations(['SET_QUOTA']),
 
+    async fileDropped(fileIdTarget) {
+      const selected = this.selectedFiles
+      const targetInfo = this.activeFiles.find(e => e.id === fileIdTarget)
+      const targetSelected = selected.some(e => e.id === fileIdTarget)
+      if (targetSelected) return
+      if (targetInfo.type !== 'folder') return
+
+      this.loading = true
```

Comment:
If we're not reloading the folder we don't need to modify `loading` during the move operation. But I'm afraid that our store design is not performing well enough and a folder listing with 5000 resources would die hard from moving 10 files into a subfolder because of 10 render updates due to reactivity in the store... still, please give it a try. ;-)